community/setup_in_windows.md

```diff
@@ -0,0 +1,56 @@
+# Project Set-Up In Windows Environment
+Windows users who want to run the Mathesar Docker development environment in WSL are advised to clone the repository in a Linux filesystem. 
+If you have tried running the project in windows you many ran into some issues including this 
+```
+service_1  | /usr/bin/env: ‘bash\r’: No such file or directory
+service_1  | 2022/03/30 16:20:21 Command exited with error: exit status 127
+```
+The complete guide is given below .
+## Pre-requisite
+- We assume you have Windows 10 or higher.
+- [Docker](https://docs.docker.com/get-docker/) is up and running properly in your system.
+
+# Solution 
+ - First search for "Turn Windows features on or off" , navigate to buttom and click the check-box named "Windows Subsystem For Linux".
+    
+ - Now you have to install any linux distro in your windows system .
+  Go to microsorft store and install any linux distro(Ubuntu prefered) and launch .
+ - If you do ls -la in ubuntu terminal you can see the linux files structure,
+     * so you have to access  window's files from ubuntu console in order to reach your destination folder in windows, use the  command given below to display all the storage drives avilable in windows , it will mount the storage .
+     
+     ``
+     cd /mnt/
+     ``
+     * navigate to your desired file location e.g 
+     ```
+     cd /mnt/User/your_pc_name/Desktop/folder
+     ```
+ - Clone the repository using command
+    ``` 
+    git clone https://github.com/centerofci/mathesar.git
+    ```
+    * Here you may run into some problems regarding permissions in some .git-config files. It may occur as Ubuntu is not configured properly in your Windows machine. Restart your machine to fix this issue.
```

Comment:
```suggestion
    * Here you may run into some problems regarding permissions in some `.git-config` files. It may occur as Ubuntu is not configured properly in your Windows machine. Restart your machine to fix this issue.
```